packages/fe-fpm-writer/test/integration/index.test.ts

```diff
@@ -69,10 +91,33 @@ describe('use FPM with existing apps', () => {
         });
 
         test('generateCustomView in ListReport', () => {
+            //pre-requisite is at least one view based on annotations
+            fs.extendJSON(join(targetPath, 'webapp/manifest.json'), {
```

Comment:
why is this extension needed? the test should also run fine when no view exists yet (but we need to test both scenarios)

Reply:
No, it does not. You cannot add a custom view to a ListReport with at least one annotation based view.

Reply:
We should consider adding this default view in such case but that would be a different topic and should be discussed with @johannes-kolbe as well

Reply:
OK, got it